src/wp-includes/functions.php

```diff
@@ -34,12 +34,12 @@ function mysql2date( $format, $date, $translate = true ) {
 
 	$datetime = date_create( $date, wp_timezone() );
 
-	if ( false === $datetime ) {
+	if (false === $datetime) {
 		return false;
 	}
 
 	// Returns a sum of timestamp with timezone offset. Ideally should never be used.
-	if ( 'G' === $format || 'U' === $format ) {
+	if ( 'G'===$format || 'U' === $format ) {
```

Comment:
```suggestion
	if ( 'G' === $format || 'U' === $format ) {
```

Example of a suggestion that can be committed from web UI (can also be batched with other suggestions).